MIDAS/src/errors.h

```diff
@@ -3,5 +3,6 @@
 
 enum ErrorCode {
     NoError,
-    LowGCouldNotBeInitialized
+    LowGCouldNotBeInitialized,
+    CANNOT_INIT_KX134_CS,
```

Comment:
Try to copy the same format as LowGCouldNotBeInitialized. Also did you mean to leave the cs in there